site/docs/messaging/quickStart.mdx

```diff
@@ -58,12 +58,13 @@ More information about sending messages can be found on our [Send Messages](/doc
 ### Sent Message Callbacks
 After sending a message from a Bandwidth number, our API will send status callbacks to your configured `OutboundCallbackURL`.
 You can configure the types of status updates you receive in the Dashboard UI and via the Dashboard API.
-These status callbacks can be handled in your local server, or with Pipedream.
+These status callbacks can be handled in your local server, or with [Pipedream][].
 More information about the status callbacks can be found on the Messaging [Webhooks](/docs/messaging/webhooks#outbound-message-webhooks) page.
 
 ## Receiving a Message
 Bandwidth phone numbers can always receive messages, assuming SMS is enabled for that number in the Dashboard.
 When a Bandwidth number receives an SMS or MMS message, a `message-received` webhook is sent to your `InboundCallbackURL`.
-This webhook may be handled by your local server, or with Pipedream; so what you chose to do based on this webhook is up to you.
+This webhook may be handled by your local server, or with [Pipedream][]; so what you chose to do based on this webhook is up to you.
 Examples for Inbound webhooks may be found on the Messaging [Webhooks](/docs/messaging/webhooks#inbound-message-webhooks) page.
 
+[Pipedream]: <https://pipedream.com/>
```

Comment:
Whaaaaaaaat???? Ignore my previous comment then. Is this an [extra-special version of markdown thing] or a [github-flavored-markdown thing]?